philips_hue.py

```diff
@@ -119,7 +119,7 @@ def trigger_light_from_monitoring_notification(client, notification, light_id):
         BadAPIRequestError: If there was an error in calling the Philips Hue API.
     """
     try:
-        condition_state = notification["incident"]["condition"]["state"]
+        condition_state = notification["incident"]["state"]
```

Comment:
condition_state is an inaccurate_name now?

Reply:
Done, I just changed the name from condition_state to incident_state and modified the code and errors accordingly.